src/services/apiService.js

```diff
@@ -194,7 +194,7 @@ export async function ConnectUserCommentsAccount(data, token) {
       message: 'Data is empty or in the wrong format',
     };
   }
-  const response = await fetch(`${API_URL}/ConnectUserCommentsFunction`, {
+  const response = await fetch(`${API_URL}/ConnectUserToCommentsFunction`, {
```

Comment:
Oh interesting! I think we should try keep the names the same. (on the Azure Function side). I think this may have been my fault 🤦

1. Please update the name back to
`ConnectUserCommentsFunction`